inc/ThirdParty/Themes/MinimalistBlogger.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Themes;
+
+class MinimalistBlogger extends ThirdpartyTheme {
+
+	/**
+	 * Name from the theme.
+	 *
+	 * @var string
+	 */
+	protected static $theme_name = 'minimalist-blogger';
+
+	/**
+	 * Return an array of events that this subscriber wants to listen to.
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		$events = [];
+		if ( self::is_theme() ) {
+			$events['rocket_delay_js_exclusions'] = 'exclude_jquery_from_delay_js';
+		}
+		return $events;
```

Comment:
Applying returning early pattern to be something like that, what do u think?

```suggestion
		if ( ! self::is_theme() ) {
			return [];
		}
		
		return [
			'rocket_delay_js_exclusions' => 'exclude_jquery_from_delay_js',
		];
```